eland/tests/dataframe/test_utils_pytest.py

```diff
@@ -65,6 +65,40 @@ def test_generate_es_mappings(self):
 
         assert_pandas_eland_frame_equal(df, ed_df_head)
 
+        ES_TEST_CLIENT.indices.delete(index=index_name)
+
+    def test_pandas_to_eland_ignore_index(self):
```

Comment:
This unit test passes without the `ignore_index=True`, should add a that fails if not using the functionality.